test/functional/testUtil/diffPrinter.js

```diff
@@ -0,0 +1,111 @@
+class DiffPrinter {
+  /* eslint-disable lodash/prefer-constant */
+  static get ANSI_RED() { return '\u001B[31m'; }
+  static get ANSI_GREEN() { return '\u001B[32m'; }
+  static get ANSI_GREY() { return '\u001B[90m'; }
+  static get ANSI_RESET() { return '\u001B[0m'; }
```

Comment:
> ```
> /* eslint-disable lodash/prefer-constant */
> ```

These can actually just be constants then? :P No good reason to put them as methods for now.